Taskfile.yml

```diff
@@ -217,8 +217,44 @@ tasks:
         -target-url=mongodb://127.0.0.1:47017/
         -target-backend=mongodb
 
+  bench-pg:
```

Comment:
Let's move all those new targets to `integration/Taskfile.yml` because that file is already too large